docs/website/docs/building-from-source/getting-started.md

```diff
@@ -44,8 +55,28 @@ git submodule update --init
 
 Configure then build all targets using CMake:
 
+Configure CMake:
+
+``` shell
+# Recommended for simple development using clang and lld:
+cmake -GNinja -B ../iree-build/ -S . \
+    -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+    -DIREE_ENABLE_ASSERTIONS=ON \
+    -DCMAKE_C_COMPILER=clang \
+    -DCMAKE_CXX_COMPILER=clang++ \
```

Comment:
(arg, GitHub ate my comment, retyping...)

We still want to recommend MSVC on Windows, right?

Could put this under a `=== "Linux and MacOS"` section or use the env var approach like before (suggest setting the env var above)
```
    export CC=clang
    export CXX=clang++
```

Reply:
Good point. I can add a tab thingy for windows. I work on a lot of different projects and the env vars are killing me.